src/Metadata/ApiResource.php

```diff
@@ -147,6 +147,7 @@ public function __construct(
         protected ?array $graphQlOperations = null,
         $provider = null,
         $processor = null,
+        protected ?string $entityClass = null,
```

Comment:
Shouldn't we create a dedicated attributes in the Doctrine namespace instead of adding Doctrine-specific properties in these class?

Reply:
We already have a bunch of vendor-specific properties (for example ES, mongodb etc). I think that its not a big deal as this class is our interface for developers. Were you thinking about some `Doctrine/Metadata/ApiResource`? but then we need to duplicate every operation?